### PR TITLE
Added cyrus-sasl

### PR DESCRIPTION
**Pull request by sherman-the-tank**

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: project/externals/cyrus-sasl.cmake [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
```
Comment (panda-sheep) [inline project/externals/cyrus-sasl.cmake:4]: I noticed that the Copyright of this repo is not up-to-date. Does it need to be consistent with the repo of `nebula`?

Comment (sherman-the-tank) [reply project/externals/cyrus-sasl.cmake:4]: Nice catch. It should be 2021 here

Comment (sherman-the-tank) [reply project/externals/cyrus-sasl.cmake:4]: Nice catch. It should be 2021 here

Comment (sherman-the-tank) [reply project/externals/cyrus-sasl.cmake:4]: Also need to remove the CC 1.0 part

Comment (panda-sheep) [reply project/externals/cyrus-sasl.cmake:4]: > Also need to remove the CC 1.0 part
ok，I'll deal with it later. ☺️

**Pull request merged**